PROTOCOL.md

```diff
@@ -2807,6 +2938,114 @@ nullCount | The number of `null` values for this column
 minValues | A value smaller than all values present in the file for this column
 maxValues | A value larger than all values present in the file for this column
 
+## API Response Format in Delta
+This section discusses the API Response Format in Delta returned by the server. When a table is shared
+as delta format, the actions in the response could be put in a delta log in the local storage on the
+recipient side for the delta library to read data out of it directly, in this way, advanced delta
+features are transparent to delta sharing, and the code duplication for the new delta features won't
+be necessary.
+
+### JSON Wrapper Object In Each Line in Delta
+
+The JSON object in each line is a wrapper object that may contain the following fields. For each
+field, it is a wrapper of a delta action(which keeps the action in its delta format and with original
+values), and with some additional fields for delta sharing functionalities.
+
+Field Name | Data Type | Description | Optional/Required
+-|-|-|-
+protocol | The [Protocol in Delta Format](#protocol-in-delta-format) JSON object. | A wrapper of delta protocol. | Optional
+metaData | The [Metadata in Delta Format](#metadata-in-delta-format) JSON object. | A wrapper of delta metadata, including some delta sharing specific fields. | Optional
+file | The [File in Delta Format](#file-in-delta-format) JSON object. | A wrapper of a delta single action in the table. | Optional
+
+It must contain only **ONE** of the above fields.
+
+### Protocol in Delta Format
```

Comment:
For all these fields what we wrap, consider adding links to the corresponding delta doc.

Reply:
added.